generators/app/templates/_package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "<%= projectNameSlugified %>",
+  "title": "<%= projectName %>",
+  "description": "<%= projectDescription %>",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "<%= repoUrl %>"
+  },
+  "dependencies": {
+    <% if (includeAphrodite && !includeRadium) {%>
+      "aphrodite": "^1.2.3",<%
+    }
+
+    if (!includeAphrodite && includeRadium) {%>
+    "radium": "^0.19.4",<%
+    }
+
+    if (includeInext) {%>
+    "i18next": "^9.0.0",<%
+    }
+
+    if (includeMobileDetect) {%>
+    "mobile-detect": "^1.3.7",<%
+    }
+
+    if (includeMoment) {%>
+    "moment": "^2.18.1",<%
+    }
+
+    if (includeNukaCarousel) {%>
+    "nuka-carousel": "^2.3.0",<%
+    }
+
+    if (includeNumeral) {%>
+    "numeral": "^2.0.6",<%
+    }
+
+    if (includePostcss) {%>
+    "postcss": "^6.0.11",<%
+    }
+
+    if (includeReactAlert) {%>
+    "react-alert": "^2.3.0",<%
+    }
+
+    if (includeReactModal) {%>
+    "react-modal": "^2.3.2",<%
+    }
+
+    if (includeReactResponsive) {%>
```

Comment:
I would remove this from the selection. This is not used very often, I would use that library only  when we need it or directly define a utils function to determine a phone width and apply decitions based on that